### PR TITLE
fix: making it compatible to newer version of @asyncapi/react-component

### PR DESCRIPTION
**Pull request by catosaurusrex2003**
**Description**
Should be merged only after [https://github.com/asyncapi/asyncapi-react/pull/1078](https://github.com/asyncapi/asyncapi-react/pull/1078) is merged.
1. Will need to change the version of `@asyncapi/react-component`

Creating an immature PR for tracking purposes

**Related issue(s)**
[https://github.com/asyncapi/asyncapi-react/pull/1078](https://github.com/asyncapi/asyncapi-react/pull/1078)
#200

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,7 +103,6 @@
     "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
   },
   "devDependencies": {
-    "@asyncapi/react-component": "^2.0.0",
```
Comment (ivangsa) [inline package.json:106]: I guess you have some special setup to remove this.. bc it's not going to work otherwise

Comment (catosaurusrex2003) [reply package.json:106]: yes, i am waiting for my pr in `@asyncapi/react-component` to get merged. Then I will update the version according to it and push it. This shouldnt be merged untill then.

Comment (catosaurusrex2003) [reply package.json:106]: @ivangsa ptal.

**Comment by catosaurusrex2003** [comment]
@ivangsa 
added the newest version of `@asyncapi/react-component`
ptal

**Review by ivangsa — APPROVED** [review]
LGTM! Thanks a lot for this @catosaurusrex2003 

I have updated the title of the PR so this is deployed as a patch version

**Pull request merged**